msg/RadarReturn.msg

```diff
@@ -1,6 +1,8 @@
 # All variables below are relative to the radar's frame of reference.
 # This message is not meant to be used alone but as part of a stamped or array message.
 
+builtin_interfaces/Duration stamp_offset # Offset to the stamp in the RadarScan message.
```

Comment:
Define this better. offset representing what? The time it takes for the total scan to be accumulated? The offset due to time sychronization? etc

Reply:
This field should have the same meaning as `time_increment` of `sensor_msgs/LaserScan`.
The `stamp_offset` should be the offset to the time stamp of the first measurement on a collected scan which is defined in the `header` in the `RadarScan` message.

When you have a rotating radar scanner, a `RadarScan` would consist on a collection of `RadarReturn`s that are from different measurements.
With this change the returns can have a precise time stamp.

---

I wanted to add an alternative description to this reply.
But I have problems finding other words to describe it (at least not in a substantial way), because I think it is already quite clear.
The ambiguity of the meaning of this sentence might be more introduced by that this message is not supposed to be used by itself, so it needs to be seen as a member of `radar_msgs/RadarScan` which time stamp is referenced in the comment.

What do you think about the following description?
It basically says the same, but might does it in a clearer way?

```suggestion
builtin_interfaces/Duration stamp_offset # The offset relative to the time stamp set in the header of the outer message
                                         # containing a collection of these returns.
```

Reply:
This needs to actually describe what its representing. You mention its the offset from the last point in the measurement for the first (e.g. accumulation time).

Reply:
I didn't meant to referencing the previous return.
Just the time stamp of the header (of the `RadarScan` message)

So the time stamp of a `RadarReturn` would be calculated by
```
<RadarScan>.header.stamp + stamp_offset
```

So the `RadarScan` would store the "base" time stamp.
And every `RadarReturn` would store the offset to this time stamp.

Reply:
Don't explain it to me, explain it in the message precisely :-)